django_eth_events/utils.py

```diff
@@ -39,8 +40,11 @@ def normalize_address_without_0x(address) -> str:
 
 
 class JsonBytesEncoder(JSONEncoder):
-        def default(self, obj):
-            if isinstance(obj, bytes):
-                return obj.decode('ascii')
-            # Let the base class default method raise the TypeError
-            return JSONEncoder.default(self, obj)
+    def default(self, obj):
+        if isinstance(obj, HexBytes):
+            return obj.hex()
+        if isinstance(obj, bytes):
+            return obj.decode('ascii')
```

Comment:
Why do you decode as `ascii` if `bytes` but you show as `hex()` if `HexBytes`?

Reply:
Yes, I will review it.